src/components/StatsTask/InputSection/InputSection.tsx

```diff
@@ -0,0 +1,185 @@
+import { formatNumbers, FormatNumberThreshold, getScaleFromString, showPrefix } from '@consts/utils'
```

Comment:
there is one input already in the project, that looks like the one from mockups, use it instead of creating a new one (or copypasting - this code looks similar to code of the input I've wrote about)